src/client/lib/PacTerminal.ts

```diff
@@ -24,18 +25,16 @@ export class PacTerminal implements vscode.Disposable {
         // https://code.visualstudio.com/api/references/vscode-api#EnvironmentVariableCollection
         this._context.environmentVariableCollection.prepend('PATH', cliPath + path.delimiter);
 
-        this._cmdDisposables.push(vscode.commands.registerCommand('pacCLI.openDocumentation', this.openDocumentation));
-        this._cmdDisposables.push(vscode.commands.registerCommand('pacCLI.openPacLab', this.openPacLab));
-        this._cmdDisposables.push(vscode.commands.registerCommand('pacCLI.pacHelp',
-            () => PacTerminal.getTerminal().sendText("pac help")));
-        this._cmdDisposables.push(vscode.commands.registerCommand('pacCLI.pacAuthHelp',
-            () => PacTerminal.getTerminal().sendText("pac auth help")));
-        this._cmdDisposables.push(vscode.commands.registerCommand('pacCLI.pacPackageHelp',
-            () => PacTerminal.getTerminal().sendText("pac package help")));
-        this._cmdDisposables.push(vscode.commands.registerCommand('pacCLI.pacPcfHelp',
-            () => PacTerminal.getTerminal().sendText("pac pcf help")));
-        this._cmdDisposables.push(vscode.commands.registerCommand('pacCLI.pacSolutionHelp',
-            () => PacTerminal.getTerminal().sendText("pac solution help")));
+        this._cmdDisposables.push(
```

Comment:
another refactor opportunity for future PR: keep the terminal specifics here, but have separate modules for command registration and panel registration